### DISCUSSION TITLE
Whether the definition of the g_dgdr_gauserf function needs to be adjusted？

### DISCUSSION THREAD

**Discussion by ExuberantWitness** [byuflowlab/FLOWVPM.jl#8] [Q&A]
While comparing the code to the corresponding paper, I seem to have noticed that "g_dgdr_gauserf" has one Sigma less than the corresponding formula. I wonder if this will affect the final result
![image](https://user-images.githubusercontent.com/30213103/199187826-702ce855-defd-4d61-a559-cdc405831b82.png)
![image](https://user-images.githubusercontent.com/30213103/199187897-aaacedf0-b103-498f-82db-e313182ce532.png)

**Comment by EdoAlvarezR** [comment]
Better late than never! =]

The second output of `g_dgdr_gauserf(...)` later on gets divided by `sigma*r` to end up obtaining the equation that you show. The mismatch between the implementation and the dissertation might be a little confusing, but that way we were able to generalize the kernel (we can switch to any kernel while keeping the same [`UJ` function](https://github.com/byuflowlab/FLOWVPM.jl/blob/4fd3aa07a37476ba2241b70ed5049d12c965b890/src/FLOWVPM_UJ.jl#L44-L104))


![image](https://github.com/byuflowlab/FLOWVPM.jl/assets/14857108/ccaaf74d-4fe3-4fd5-afc1-fc826f0f5f1d)
